args.go

```diff
@@ -429,8 +429,8 @@ const (
 	ArgDatabaseTopicMaxCompactionLagMS = "max-compaction-lag-ms"
 	// ArgDatabaseTopicMaxMessageBytes is the maximum size, in bytes, of the largest record batch that can be sent to the server
 	ArgDatabaseTopicMaxMessageBytes = "max-message-bytes"
-	// ArgDatabaseTopicMesssageDownConversionEnable determines whether brokers should convert messages for consumers expecting older message formats
-	ArgDatabaseTopicMesssageDownConversionEnable = "message-down-conversion-enable"
+	// ArgDatabaseTopicMessageDownConversionEnable determines whether brokers should convert messages for consumers expecting older message formats
+	ArgDatabaseTopicMessageDownConversionEnable = "message-down-conversion-enable"
```

Comment:
As this is a "code aware" spell checker, any  chance you happen to know if there is a way to exclude existing exported vars/consts? That's not really an issue for doctl itself, but if we were to bring this into say godo, we'd want to avoid breaking changes.

Reply:
so best i've found in the past was `default.extend-identifiers` in the config, but nothing automated
I believe typos also has a cli flag for listing all the identifiers it finds

that being said, fixing the spelling is an easy breaking change to handle for end users, its not like they have to figure out how to rewrite code because a function no longer exists.